src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -628,9 +633,10 @@ func (s *service) fetchTagged(ctx context.Context, db storage.Database, req *rpc
 	nsID := results.Namespace()
 	nsIDBytes := nsID.Bytes()
 
-	// NB(r): Step 1 if reading data then read using an asychronuous block reader,
+	// NB(r): Step 1 if reading data then read using an asynchronous block reader,
 	// but don't serialize yet so that all block reader requests can
 	// be issued at once before waiting for their results.
+	ctx, sp := ctx.StartTraceSpan(tracepoint.FetchReadEncoded)
```

Comment:
Is there a reason this one is unsampled?

Reply:
The method name here is a bit misleading, but this still has sampling applied. The difference is just that this is a wrapper method around the `StartSampledTraceSpan` and it drops the `sampled` return value.

```
func (c *ctx) StartTraceSpan(name string) (Context, opentracing.Span) {
	ctx, sp, _ := c.StartSampledTraceSpan(name)
	return ctx, sp
}
```